mypy/nodes.py

```diff
@@ -689,6 +690,11 @@ def __init__(self,
     def name(self) -> str:
         return self._name
 
+    @property
```

Comment:
Let's not add this property:
 1. Nothing in `mypy` currently imports `mypyc` and I think we should keep it that way
 2. I don't want to introduce a new `id` concept to mypy itself